crates/turborepo-lib/src/cli.rs

```diff
@@ -60,7 +60,7 @@ pub struct Args {
     #[clap(long, global = true)]
     pub color: bool,
     /// Specify a file to save a cpu profile
-    #[clap(long, global = true, value_parser)]
+    #[clap(long, aliases = ["cpuprofile"], global = true, value_parser)]
```

Comment:
Should we just switch the name back to `cpuprofile`?
```suggestion
    #[clap(long = "cpuprofile", global = true, value_parser)]
```

Reply:
Yea that works, cpu-profiler is cleaner imo. But there's a few other args we should rename for clarity so maybe we can do them all a once when 2.0 comes around